### PR TITLE
Add console warnings if a shortcut has been skipped.

### PR DESCRIPTION
**Pull request by rayankans**
Resolves https://github.com/GoogleChromeLabs/bubblewrap/issues/116

### CODE REVIEW

**Comment by rayankans** [comment]
@andreban Do you mind taking a look

**Comment by andreban** [comment]
Have you considered using [`Log.ts`](https://github.com/GoogleChromeLabs/bubblewrap/blob/master/packages/core/src/lib/Log.ts) to output the warnings? There's a `Log.warn()` method that will format it as a warning (yellow) and with a tag.

**Comment by rayankans** [comment]
Changed this to use Log.ts

File: packages/core/src/lib/TwaManifest.ts [hunk 1]
```diff
@@ -100,6 +101,8 @@ export class TwaManifest {
   generatorApp: string;
   webManifestUrl?: URL;
 
+  private static log: Log = new Log('twa-manifest');
```
Comment (andreban) [inline packages/core/src/lib/TwaManifest.ts]: nit: I've been passing a log object in the constructor, but initialising with a default value. This helps with tests so those don't pollute the console. But I'm ok to keep like this too.

Comment (rayankans) [reply packages/core/src/lib/TwaManifest.ts]: It's being used from within a static function though. I could make it a module-level constant but that won't help with the pollution issue.

**Pull request merged**